scaleway/resource_instance_server.go

```diff
@@ -508,6 +508,9 @@ func resourceScalewayInstanceServerUpdate(ctx context.Context, d *schema.Resourc
 
 	if d.HasChange("tags") {
 		tags := expandStrings(d.Get("tags"))
+		if tags == nil {
```

Comment:
func expandStrings(data interface{}) []string {
	if data == nil {
		return []string{}
	}
	stringSlice := []string{}
	for _, s := range data.([]interface{}) {
		stringSlice = append(stringSlice, s.(string))
	}
	return stringSlice
}